utils/http.go

```diff
@@ -0,0 +1,64 @@
+package utils
+
+import (
+  "strings"
+  "errors"
+  "net/http"
+  "io"
+  "os"
+)
+
+
+func GetRawURL(url string) string {
+	if strings.Contains(url, "github.com") {
+		raw:=strings.Replace(url, "github.com", "raw.githubusercontent.com", 1)
+		return strings.Replace(raw, "/blob", "", 1)
+	}
+	return url
+}
+
+func IsGitHubURL(url string) bool {
+	return strings.Contains(url, "github")
```

Comment:
This contains in my view is too broad. since also a "raw.githubusercontent.com" url would match.
In cmd/root.go you use this in the switch and consecutively apply the GetRawUrl. Which then does the conversion.
although that would check for "github.com" it should probably be more kind of "consistent".

Reply:
I'm not sure I'm grasping what you'd like to see from this method.
You mentioned that it would match on raw.githubusercontent.com and you're right.  I had planned on it matching on either a generic github.com or the raw.github path to offer flexibility to the end user.  It then proceeds to that GetRawURL method where it only converts it if its a generic github.com url else it returns the url without converting anything.
How would you envision this changing?